src/Reflection/ReflectionClassConstant.php

```diff
@@ -15,6 +15,10 @@
 
 class ReflectionClassConstant
 {
+    public const IS_PUBLIC    = 1;
```

Comment:
I think these shouldn't really be declared here - perhaps we should just use the built-in constants?

Reply:
I wasn't sure what to do about them for users of PHP7, but now that the filter logic is gone, so are these constants.